http/test/workflow.test.ts

```diff
@@ -0,0 +1,67 @@
+import { Connection, WorkflowClient } from '@temporalio/client';
+import { Worker, DefaultLogger } from '@temporalio/worker';
+import { describe, before, after, afterEach, it } from 'mocha';
+import { example } from '../lib/workflows';
+import assert from 'assert';
+import axios from 'axios';
+import sinon from 'sinon';
+
+describe('example workflow', function() {
+  let runPromise = null;
+  let worker = null;
+  let workflow = null;
+
+  before(async function() {
+    this.timeout(10000);
+    worker = await Worker.create({
+      workDir: `${__dirname}/../lib`,
+      taskQueue: 'testhttp',
+      logger: new DefaultLogger('ERROR'),
+    });
+
+    runPromise = worker.run();
+  });
+
+  beforeEach(function() {
+    const connection = new Connection();
+    const client = new WorkflowClient(connection.service);
+
+    workflow = client.createWorkflowHandle(example, { taskQueue: 'testhttp' });
+  });
+
+  after(async function() {
+    worker.shutdown();
+    await runPromise;
+  });
+
+  afterEach(() => {
+    axios.get.restore && axios.get.restore();
```

Comment:
```suggestion
    sinon.restore();
```

I think it's simpler and more maintainable to `sinon.restore()` in afterEach.

Reply:
Great suggestion, thanks!